hub/util/exceptions.py

```diff
@@ -419,3 +419,10 @@ def __init__(
         message="If the size of the last chunk is given, it must be smaller than the requested chunk size.",
     ):
         super().__init__(message)
+
+
+class WindowsSharedMemoryError(Exception):
+    def __init__(self):
+        super().__init__(
+            f"Python Shared memory with multiprocessing doesn't work properly on Windows."
```

Comment:
```suggestion
            f"Python Shared memory with multiprocessing doesn't work properly on Windows just yet!"
```

Reply:
Didn't want to add that as it's not upto us, it's on windows to fix its issues 😅